publiccode.yml

```diff
@@ -0,0 +1,36 @@
+publiccodeYmlVersion: "0.4"
+
+name: OSRD
+
+url: "https://github.com/OpenRailAssociation/osrd"
+
+platforms:
+  - web
+
+developmentStatus: development
+
+softwareType: "standalone/web"
+
+description:
+  en:
+    shortDescription: >
+          OSRD is an open source web application for railway infrastructure design, capacity analysis, timetabling and simulation.
+    documentation: https://osrd.fr/en/docs/
+  fr:
+    shortDescription: >
+          OSRD est une application web open source pour la conception d'infrastructures ferroviaires, l'analyse de la capacité, la programmation et la simulation.
```

Comment:
Pedantic mode enabled: this is a tad too long, spec says it's 150 characters max (and this seems to be 152).

Reply:
I didn't mean the line length here, but the full length of the `shortDescription` field: https://yml.publiccode.tools/schema.core.html#key-description-lang-shortdescription

Reply:
![face palm](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdXlhNmZ0ZWJpZjcwZHF2eWUyYWh4Z2RydnZlcWRhbGV0MndxbDVvZSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/AjYsTtVxEEBPO/giphy.gif)

Reply:
It's fixed